src/pages/downloads/index.js

```diff
@@ -96,9 +100,9 @@ export default function Downloads() {
                           href={"https://nightly.link/PGMDev/PGM/workflows/deploy/dev/PGM.jar.zip"}
                           title={"Latest development build directly from GitHub.\nContains all the latest features, may occasionally bring new bugs."}
                       >
-                        Latest <FontAwesomeIcon icon={faDownload} />
+                        Download <FontAwesomeIcon icon={faDownload} />
                       </a>
-                      <a
+                      {/* <a
```

Comment:
Dont comment out code, remove it

If you need it in the future you can get it off of git history

Reply:
All good now